docs/userguide/services.md

```diff
@@ -95,17 +97,17 @@ are represented as negative numbers.
 ICU provides the following [classes](datetime/index.md) to support calendars and
 time zones:
 
-*   [Calendar](datetime/calendar/index.md) (§)
+*   [Calendar](datetime/calendar/index.md#Calendar)
```

Comment:
I believe that the `#` anchors are lowercase with `-` separators.

Reply:
done